Moderation-Policy.md

```diff
@@ -0,0 +1,80 @@
+ * The original Post being modified (as replacement or appended content),
+ * A new post within the original thread, or
+ * A new issue within the private nodejs/moderation repository.
+* Moderation of Posts authored by non-Collaborators may result in those non-Collaborators being Banned from further participation in the Node.js Github Organization for an indefinite period of time.
+
+#### Requesting Moderation
+
+Requests to Moderate a Post are only necessary when the Post in question was written by a Collaborator.
+
+Requesting Moderation of a Post can be accomplished in one of five ways:
+
+* Private communication between the individuals involved,
+* Via the `report@io.js` email address,
+* Via private email to individual TSC members,
+* Via a new Post in the same thread as the Post being requested for Moderation, or
+* Via a new Post in the private nodejs/moderation repository.
+
+Use of the `report@io.js` email address, or private email to individual TSC members, is appropriate when the individual requesting the Moderation does not feel comfortable directly or publicly making the request. All emails sent to the `report@io.js` address are currently forwarded to all members of the TSC. [[ED NOTE: This has not yet been set up]]
+
+Requests should contain as much information and context as possible, including the URL of the Post in question.
+
+External public venues or social media services such as Twitter should never be used to request Moderation or to discuss the details of a Moderation request.
+
+### Consideration of Intent
+
+Before Moderating a Post, Collaborator's should carefully consider the possible intent of the author. It may be that the author has simply made an error or is not familiar with the Code of Conduct; or it may be that cultural differences exist or that the author is unaware that certain content is considered inappropriate. Unless the offense or the intent to disrupt, annoy or harass is obvious, an author should be given the benefit of the doubt and be given an opportunity to correct any mistake that may have been made.
+
+### Privacy of the nodejs/moderation Repository
+
+The nodejs/moderation Repository is used to discuss the potentially sensitive details of any specific proposed or actual Moderation or Banning action. The repository is private but accessible to all Collaborators. The details of any issue discussed within the nodejs/moderation repository are expected to remain confidential and are not to be discussed in any public forum or social media service.
+
+### Modifications to This Policy
+
+Modifications to this policy are made through normal [TSC motion and vote](https://github.com/nodejs/TSC/blob/master/TSC-Charter.md#section-8-voting). Any Collaborator may submit a PR proposing changes to this policy. Those PRs must be labeled using the `tsc-agenda` label. Including a mention to `@nodejs/tsc` can be used to call the issue to TSC's attention.
```

Comment:
Does that apply to the initial version of this document? It seems like it should.